gapic/utils/checks.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Google LLC
```

Comment:
Please use `field.type` instead `field.mock_value` in your test templates. By doing that you may avoid guessing work here, as we discussed over GVC. Also, please update copyright to 2020 here and in all your new files.